app/frontend/src/aquifers/components/SingleAquiferMap.vue

```diff
@@ -398,8 +431,56 @@ export default {
         canInteract,
         ecocatLayerIds: [ DATABC_ECOCAT_LAYER_ID ]
       })
+    },
+    updateMapLegendBasedOnVisibleElements() {
+      //gets a list of rendered objects and then updates the legend to only display entries for items that are currently rendered
+      const uniqueRenderedLayerIds = this.getRenderedLayerIds();
```

Comment:
Can we use proper documentation for functions instead of a bunch of single line comments
```js
    },
    /**
    * @desc <information>
    */
    updateMapLegendBasedOnVisibleElements() {
      //gets a list of rendered objects and then updates the legend to only display entries for items that are currently rendered
      const uniqueRenderedLayerIds = this.getRenderedLayerIds();
```

Reply:
really good point. done